src/setup/devnet/templates/remote/service-host.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env sh
+
+NODE_DIR=$HOME/node
+BOR_HOME=$HOME/.bor
+BIN_DIR=$(go env GOPATH)/bin
+USER=$(whoami)
+
+
+VALIDATOR_ADDRESS="`cat $BOR_HOME/address.txt`"
+
+cat > metadata <<EOF
+VALIDATOR_ADDRESS=
+EOF
+
+cat > ganache.service <<EOF
+[Unit]
+    Description=ganache
+[Service]
+    WorkingDirectory=$HOME
+    Environment=PATH=/home/ubuntu/.nvm/versions/node/v10.17.0/bin:/home/ubuntu/go/bin:/home/ubuntu/.go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
+    ExecStart=/bin/bash $HOME/ganache-start-remote.sh
+    User=ubuntu
+    Type=simple
+    KillSignal=SIGINT
+EOF
+
+cat > bor.service <<EOF
+[Unit]
+  Description=bor
+  StartLimitIntervalSec=500
+  StartLimitBurst=5
+[Service]
+  Restart=on-failure
+  RestartSec=5s
+  WorkingDirectory=$NODE_DIR
+  Environment=PATH=/home/ubuntu/.nvm/versions/node/v10.17.0/bin:/home/ubuntu/go/bin:/home/ubuntu/.go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
```

Comment:
can we replace `/home/ubuntu` with `$HOME` ? So we are not dependant on users' names